### ISSUE TITLE
Error on building HTML5

### ISSUE THREAD

**Issue by PatoFlamejanteTV** [JordanSantiagoYT/FNF-JS-Engine#727]
### Describe your bug here. If you are modding, try testing your bug in a clean version of the engine instead. Also, if you're using an older version of JSE, please try the latest version/action build. Also, be sure to check the pinned 'JS Engine: Known Issues' issue, to check if your issue hasn't already been found!

Hi! I was trying to compile the JS Engine (no mods and no modifications to the source code) to HTML, but I keep getting this error:

```
C:/HaxeToolkit/haxe/lib/linc_luajit/git/llua/State.hx:11: characters 24-49 : You cannot access the cpp package while targeting js (for cpp.RawPointer)
```
And I cant play the exported version because it needs the `.js` output file.

### Command Prompt/Terminal/Crash logs (if existing)

PS C:\Users\mcmco\Downloads\FNF-JS-Engine-main> haxelib run lime test html5 -release
C:/HaxeToolkit/haxe/lib/linc_luajit/git/llua/State.hx:11: characters 24-49 : You cannot access the cpp package while targeting js (for cpp.RawPointer)
C:/HaxeToolkit/haxe/lib/linc_luajit/git/llua/Lua.hx:4: characters 8-18 : ... referenced here
C:\Users\mcmco\Downloads\FNF-JS-Engine-main\source\import.hx:14: characters 8-16 : ... referenced here
C:\Users\mcmco\Downloads\FNF-JS-Engine-main\source\import.hx:2: characters 8-13 : ... referenced here
export/release/html5/haxe/ApplicationMain.hx:321: characters 48-52 : ... referenced here

### Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.

```
PS C:\Users\mcmco\Downloads\FNF-JS-Engine-main> haxelib run lime test html5 -release

    Directory: C:\Users\mcmco\Downloads\FNF-JS-Engine-main


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         11/6/2024   1:06 PM                .github
d-----         11/6/2024   1:06 PM                art
d-----         11/6/2024   1:06 PM                assets
d-----         11/6/2024   1:06 PM                docs
d-----         11/6/2024   1:06 PM                example_mods
d-----         11/6/2024   4:25 PM                export
d-----         11/6/2024   1:06 PM                setup
d-----         11/6/2024   1:06 PM                source
------         11/6/2024   1:06 PM             66 .gitattributes
------         11/6/2024   1:06 PM            567 .gitignore
------         11/6/2024   1:06 PM           3440 BUILDING.md
------         11/6/2024   1:06 PM           1879 hmm.json
------         11/6/2024   1:06 PM            250 hxformat.json
------         11/6/2024   1:06 PM          11324 LICENSE
------         11/6/2024   1:06 PM           7446 Project.xml
------         11/6/2024   1:06 PM           8306 README.md
------         11/6/2024   1:06 PM            710 THECHANGELOG.md
------         11/6/2024   1:06 PM            188 UPDATE HAXE TO 4.2.5.txt
```

### Are you modding a build from source or with Lua?

Source

### What is your build target?

HTML5

### Did you edit anything in this build? If so, mention or summarize your changes.

No modifications to the source code

### If you use 1.12.0 or earlier, did you have Optimized Chart Loading turned on?

None

### Did you check for any similar issues to what you're reporting? Check, then come back here. If there is a similar issue, then do not report the issue, otherwise it will be marked as a duplicate.

Yes

**Comment by mcagabe19** [comment]
I don't think jordan supports with html5 but it's all goes to Project.xml psych engine defines, look at them

**Comment by PatoFlamejanteTV** [comment]
> I don't think jordan supports with html5 but it's all goes to Project.xml psych engine defines, look at them

ok

**Comment by PatoFlamejanteTV** [comment]
> I don't think jordan supports with html5 but it's all goes to Project.xml psych engine defines, look at them

Ok I spent hours looking and didn't find out why this was having problems.

**Comment by moxie-coder** [comment]
I've mostly addressed this in the recent commit 8bf331a, which *kinda* fixes some problems, but not quite all of them though

**Comment by StinkTheStinker** [comment]
close for inactivity
